### ISSUE TITLE
Tags look odd in kubectl describe

### ISSUE THREAD

**Issue by randomvariable** [kubernetes-sigs/cluster-api-provider-aws#260]
/kind bug

**What steps did you take and what happened:**
This was also an issue pre-CRD, but running `kubectl describe` returns a tag field in the cluster status that looks like this:

```
Tags:
Name:                                                          test1-bastion
Kubernetes . Io / Cluster / Test 1:                            owned
Sigs . K 8 S . Io / Cluster - API - Provider - Aws / Managed:  true
Sigs . K 8 S . Io / Cluster - API - Provider - Aws / Role:     bastion
```

**What did you expect to happen:**
Tags to be:

```
sigs.k8s.io/cluster-api-provider-aws/role:          bastion
sigs.k8s.io/cluster-api-provider-aws/managed: true
kubernetes.io/cluster/test1:                                owned
```

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster-api-provider-aws version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

**Comment by vincepri** [comment]
Is this an issue with kubectl or with our tag system?

**Comment by randomvariable** [comment]
Not sure if it's kubectl trying to auto format delimiters. Might be a way to stop it if so?
Forgot the kubectl version, in any case, it's the latest binaries, and this was against a 1.12.1 API server too.
```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.1", GitCommit:"4ed3216f3ec431b140b1d899130a69fc671678f4", GitTreeState:"clean", BuildDate:"2018-10-05T16:46:06Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
```

Haven't investigated too deeply - saw it pre alpha v1 but it wasn't breaking anything, so left it.

**Comment by timothysc** [comment]
@vincepri - could you try to repro.

**Comment by vincepri** [comment]
This seems like a kubectl printing issue more than tags being stored improperly. Running describe in verbose mode, which prints requests/reponses the data looks fine.

**Comment by gsadhani** [comment]
@vincepri I looked at the issue and was able to root cause it to kubectl output formatting. I am new to cluster-api-provider-aws, can I take this up?

**Comment by detiber** [comment]
@gsadhani yes, I will go ahead and set the lifecycle active flag to let others know it is being worked on. If you run into any issues, or decide to no longer work on it please let me know.

/assign
/lifecycle active

**Comment by detiber** [comment]
This will be fixed when interacting with kubectl >= v1.15
